src/main/java/org/prebid/server/privacy/PrivacyExtractor.java

```diff
@@ -71,33 +70,33 @@ private Privacy extractPrivacy(Regs regs, User user, List<String> errors) {
         final String usPrivacy = extRegs != null ? extRegs.getUsPrivacy() : null;
         final Integer coppa = regs != null ? regs.getCoppa() : null;
 
-        return toValidPrivacy(gdpr, consent, usPrivacy, coppa, errors);
+        return toValidPrivacy(gdpr, consent, usPrivacy, coppa, prebidLog);
     }
 
     private static Privacy toValidPrivacy(String gdpr,
                                           String consent,
                                           String usPrivacy,
                                           Integer coppa,
-                                          List<String> errors) {
+                                          PrebidLog prebidLog) {
         final String validGdpr = ObjectUtils.notEqual(gdpr, "1") && ObjectUtils.notEqual(gdpr, "0")
                 ? DEFAULT_GDPR_VALUE
                 : gdpr;
         final String validConsent = consent == null ? DEFAULT_CONSENT_VALUE : consent;
-        final Ccpa validCcpa = usPrivacy == null ? DEFAULT_CCPA_VALUE : toValidCcpa(usPrivacy, errors);
+        final Ccpa validCcpa = usPrivacy == null ? DEFAULT_CCPA_VALUE : toValidCcpa(usPrivacy, prebidLog);
         final Integer validCoppa = coppa == null ? DEFAULT_COPPA_VALUE : coppa;
 
         return Privacy.of(validGdpr, validConsent, validCcpa, validCoppa);
     }
 
-    private static Ccpa toValidCcpa(String usPrivacy, List<String> errors) {
+    private static Ccpa toValidCcpa(String usPrivacy, PrebidLog prebidLog) {
         try {
             Ccpa.validateUsPrivacy(usPrivacy);
             return Ccpa.of(usPrivacy);
         } catch (PreBidException e) {
             final String message = String.format("CCPA consent %s has invalid format: %s", usPrivacy, e.getMessage());
             logger.debug(message);
-            if (errors != null) {
-                errors.add(message);
+            if (prebidLog != null) {
```

Comment:
Seems this check is redundant

Reply:
It's not, was discussed in a call with you in it :)